src/demo/demo.test.js

```diff
@@ -34,30 +33,33 @@ describe('Mount Demo App', () => {
     rootNode.setAttribute('id', 'demo-app')
     window.domNode = rootNode
     document.body.appendChild(rootNode)
-
-    Demo = require('./demo.js')
   })
 
   describe('by mocking Onfido SDK', () => {
-    // Mock window.Onfido
-    global.Onfido = {
-      init: () => {},
-    }
+    beforeEach(() => {
+      // Mock window.Onfido
+      window.Onfido = {
+        init: jest.fn(),
+      }
+
+      Demo = require('./Demo').Demo
+    })
 
     it('mounts the Onfido Demo without crashing', () => {
-      // the component needs to be assigned to a lowercase variable to work!
-      const sdk = <Demo />
-      const sdkDemo = mount(<sdk />)
-      expect(sdkDemo.exists()).to.equal(true)
+      const sdkDemo = mount(<Demo />)
+      expect(sdkDemo.exists()).toBeTruthy()
+      expect(window.Onfido.init).toHaveBeenCalled()
     })
   })
 
   describe('without mocking Onfido SDK', () => {
+    beforeEach(() => {
+      Demo = require('./Demo').Demo
+    })
+
     it('mounts the Onfido Demo without crashing', () => {
-      // the component needs to be assigned to a lowercase variable to work!
-      const sdk = <Demo />
-      const sdkDemo = mount(<sdk />)
-      expect(sdkDemo.exists()).to.equal(true)
+      const sdkDemo = mount(<Demo />)
```

Comment:
I found some problems related to our unit test setup, which didn't properly trigger the hooks inside `<Demo />` and `<SDK />` components. What I've changed:
- Expose `Demo` component for direct imports
- Mount `<Demo />` component instead of `<sdk />` for Enzyme, which intuitively didn't make sense from the beginning.
- Prevent rendering execution (`renderDemoApp()`) when `process.env.NODE_ENV=test`
- (Minor) Use `jest.expect()` instead of `chai.expect()`

Let's re-review @stefaniacardenas @kopijunkie

Reply:
Why do you want to "Prevent rendering execution (renderDemoApp()) when process.env.NODE_ENV=test"?

Reply:
The initial attempt is to avoid duplicated renders when import from unit test files, but after giving it another try, it seems that the `renderDemoApp()` doesn't affect